docs/dev/libvirt/README.md

```diff
@@ -286,6 +286,38 @@ kubectl get --all-namespaces pods
 ## Troubleshooting
 If following the above steps hasn't quite worked, please review this section for well known issues.
 
+### Console doesn't come up
+In case of libvirt there is no wildcard dns resolution and console depends on the route which is created by auth operator.
+To make it works we need to first create the manifests and edit the `domain` for ingress config, before directly creating the cluster.
+
+Following steps are required to have console operator up and working.
+```console
+// Step-1
```

Comment:
nit: `//` are not shell comments.  Either use `$ # Step-1`, etc., or (my preference), shift this supporting text outside of the fenced code block:

    Add another domain entry in `openshift.conf`, which is...

    ```console
    $ cat /etc/NetworkManager/dnsmasq.d/openshift.conf
    server=/tt.testing/192.168.126.1
    address=/.apps.tt.testing/192.168.126.51
    ```

    Make sure you restart ...